builder/src/permissioned.rs

```diff
@@ -384,7 +386,13 @@ pub async fn init_hotshot<
             .unwrap(),
         ConsensusMetricsValue::new(metrics),
         da_storage,
-        TestAuctionResultsProvider::default(),
+        // TODO if this builder is to be used for marketplace, we need real urls.
```

Comment:
It is not, we can just pass bogus config.

Reply:
removed c5bb9849f86685a5a58e8c6d48e3190b4fc9e564